### ISSUE TITLE
Keybase is a greA

### ISSUE THREAD

**Issue by wayneroberts386** [keybase/keybase-issues#2469]

**Comment by zQueal** [comment]
![uahufd](https://cloud.githubusercontent.com/assets/239193/17463835/c0f9d3bc-5c9d-11e6-8a3d-ea1f85e31b86.png)

**Comment by wayneroberts386** [comment]
Yeah I spotted that, and followed the link, if I am looking at the right page, there are four entries.
Has any member volunteered to write some sort of how-to pages? If not I would like to volunteer.

**Comment by tehbmar** [comment]
```
keybase help
keybase help <command>
```
Also
https://keybase.io/docs

The CLI documentation for the most part is self writing but I could see the docs needing contributing.  But they probably wont take any volunteers on until the Alpha is nearer to completion (Just my speculation though)
